components/script/dom/dedicatedworkerglobalscope.rs

```diff
@@ -472,7 +483,12 @@ impl DedicatedWorkerGlobalScopeMethods for DedicatedWorkerGlobalScope {
         let task = Box::new(task!(post_worker_message: move || {
             Worker::handle_message(worker, data);
         }));
-        self.parent_sender.send(CommonScriptMsg::Task(WorkerEvent, task, Some(pipeline_id))).unwrap();
+        self.parent_sender.send(CommonScriptMsg::Task(
+            WorkerEvent,
+            task,
+            Some(pipeline_id),
+            TaskSourceName::DOMManipulation,
```

Comment:
Could you please add a TODO, and file a corresponding issue, pointing out that we should be using a new `unshipped-port-message-queue` task source that should be used here?

See https://html.spec.whatwg.org/multipage/#unshipped-port-message-queue (I got to this following the spec link just above the `fn PostMessage` and looking for the task source that would correspond to this specific tasks, it's often mentioned somewhere else than where the task itself is mentioned...)